tools/vagrant/vagrant-libvirt.md

```diff
@@ -18,10 +19,11 @@ There is also *@vagrant* package collection created with libvirt provider in min
 $ sudo dnf install @vagrant
 ```
 
-Afterwards make sure that libvirt daemon is running and that you have `kvm` module loaded in the kernel:
+Afterwards make sure that libvirt and virtnetworkd daemons are running and that you have `kvm` module loaded in the kernel:
```

Comment:
From how packaged libvirt daemons are right now in Fedora, the maintainer insists reboot is needed for everything to work properly.